go

```diff
@@ -64,8 +64,12 @@ def update_gems
   exec_cmd 'bundle'
 end
 
+def update_submods
+  edec_cmd "git submodule update --remote"
+end
+
 def update_data
-  exec_cmd 'cd _data && ./import-public.rb'
+  exec_cmd '_data/import-public.rb'
```

Comment:
For this to work on the remote server, I think you're gonna have to pass the path to the ruby interpreter.